### PR TITLE
Add a point cloud example that changes the point orientation

### PR DESCRIPTION
**Pull request by WeirdRubberDuck**
Note that it is really difficult to see what is going on without a texture, so added a texture to this example as well.

### CODE REVIEW

File: data/assets/examples/renderable/renderablepointcloud/pointcloud_textured_facecameraposition.asset [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+--
+-- A texture is added so that we can more easily see how the orientation is altered.
+-- See Textured example for more details.
+
+local Node = {
+  Identifier = "RenderablePointCloud_Example",
+  Renderable = {
+    Type = "RenderablePointCloud",
+    File = asset.resource("data/dummydata.csv"),
+    -- Change the oreientation render option to face the camera position instead
```
Comment (WeirdRubberDuck) [inline data/assets/examples/renderable/renderablepointcloud/pointcloud_textured_facecameraposition.asset:16]:
```suggestion
    -- Change the orientation render option to face the camera position instead
```

File: data/assets/examples/renderable/renderablepointcloud/pointcloud_textured_facecameraposition.asset [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    -- of it's view direction
```
Comment (alexanderbock) [inline data/assets/examples/renderable/renderablepointcloud/pointcloud_textured_facecameraposition.asset:17]:
```suggestion
    -- of its view direction
```

**Review by alexanderbock — APPROVED** [review]

File: data/assets/examples/renderable/renderablepointcloud/pointcloud_textured_facecameraposition.asset [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+-- oriented to face the camera position, rather than the view direction (which is
+-- default). This means that the point will be layed out in a spherical way around the
+-- camera, which works better for spherical displays compared to the default
+-- orientation.
+--
+-- A texture is added so that we can more easily see how the orientation is altered.
+-- See Textured example for more details.
+
+local Node = {
+  Identifier = "RenderablePointCloud_Example",
```
Comment (Roxeena) [inline data/assets/examples/renderable/renderablepointcloud/pointcloud_textured_facecameraposition.asset:12]: Won't this identifier collide with other examples if I were to load several of them at the same time?

**Review by Roxeena — COMMENTED** [review]
I have not been able to run this example yet but a general thought I had was that maybe we should decide on one term to use for this behavior since we have a few versions of it (billboard, face camera normal, face camera position, etc.). They all do/mean the same or similar things and it can be confusing sometimes.

File: data/assets/examples/renderable/renderablepointcloud/pointcloud_textured_facecameraposition.asset (hunk 3 above)
Comment (WeirdRubberDuck) [reply data/assets/examples/renderable/renderablepointcloud/pointcloud_textured_facecameraposition.asset:12]: Yes!  I have forgotten to change it form the example I copy-pasted ;)  Thank you

**Comment by WeirdRubberDuck** [comment]
> I have not been able to run this example yet but a general thought I had was that maybe we should decide on one term to use for this behavior since we have a few versions of it (billboard, face camera normal, face camera position, etc.). They all do/mean the same or similar things and it can be confusing sometimes.

Not really sure what you are suggesting that we change. Where should that consistent word be used? 

The word I've used in general for this parameter since adding the "Fixed Rotation" option is "orientation" or "Orientation Mode". The "Fixed Rotation" mode does not lead to billboarding, the points just have a fixed orientation in space. So, I have removed any mentioning of billboarding unless it's in the descriptions of the two orientation modes that have a "billboarding" behavior. In summary, "Camera Position Normal" and "Camera View Direction" are billboarded orientation modes, while "Fixed Rotation" is not. 

If you want to think more about this @Roxeena, I have an open PR in the Docs repo that adds documentation about these things. Would be great if you wanted to give some feedback on that :) 
https://github.com/OpenSpace/OpenSpace-Docs/pull/26

**Comment by alexanderbock** [comment]
Let's go with the different names right now and then revisit this once we have a sufficiently large number of examples.  Then we can identify multiple name changes in one go

**Pull request merged**